README.md

```diff
@@ -98,4 +98,4 @@ Please refer to our [FAQ](https://ark-analysis.readthedocs.io/en/latest/_rtd/faq
 
 ## Want to contribute?  
 
-If you would like to help make `ark` better, please take a look at our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html)
+If you would like to help make `ark` better, please take a look at our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html). You can look at our list of open issues [here](https://github.com/angelolab/ark-analysis/issues). 
```

Comment:
Sorry, I wasn't clear before. The ReadMe should have the issues link for people who need help. 

i.e. "if you run into trouble, first check out our FAQ page. If that doesn't answer you question, you can open an issue"

Reply:
Ah OK.